assets/src/img/old-rackspace-logo.svg

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Rackspace" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="186 -44 400 117" style="enable-background:new 186 -44 400 117;" xml:space="preserve">

```

Comment:
the viewbox differs between old and new logo:
old [186 -44]
new [0 0]

There seem to be other differences in how the two image sizes are specified. 
Please check with the graphic artist about the significance of these differences, if any.

https://webdesign.tutsplus.com/tutorials/svg-viewport-and-viewbox-for-beginners--cms-30844